src/network/networktool.cpp

```diff
@@ -12,39 +12,39 @@
 
 PING_LOGGING_CATEGORY(NETWORKTOOL, "ping.networktool")
 
-
-QString NetworkTool::_gitUserRepo = "bluerobotics/ping-viewer";
-QString NetworkTool::_gitUserRepoFirmware = "bluerobotics/ping-firmware";
-
+namespace {
+    QString gitUserRepo = QStringLiteral("bluerobotics/ping-viewer");
+    QString gitUserRepoFirmware = QStringLiteral("bluerobotics/ping-firmware");
+    const QString gitReleases = QStringLiteral("https://api.github.com/repos/%1/releases");
```

Comment:
maybe: `githubReleasesApiPath`to be more clear, just gitReleases does not imply that the variable is a static path for an API.